db/institutions.yml

```diff
@@ -285,3 +285,11 @@ utoronto:
   state: Ontario
   street: 27 King's College Cir
   zip: M5S 3H7
+ubelgrade:
+  aka: [University of Belgrade, Универзитет у Београду, Univerzitet u Beogradu]
+  city: Belgrade
+  country: Serbia
+  institutes: {physics: {name: Institute of Physics, aka: [IPB, Institue of Physics Belgrade]}}
```

Comment:
this should be departments.  You can't change the schema (i.e., you can't change the key names.)